include/fastdds/dds/topic/Topic.hpp

```diff
@@ -75,9 +75,6 @@ class Topic : public DomainEntity, public TopicDescription
 
 public:
 
-    /**
-     * @brief Destructor
-     */
     virtual ~Topic();
```

Comment:
Maybe we should consider making this destructor protected.

Reply:
That would break API. We may do it on a PR targetted for v3.0.